packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -76,10 +74,15 @@ function TokenListRow({ tokenList }: { tokenList: BridgeTokenList }) {
     [arbTokenBridge, token]
   )
 
-  const isActive = Object.keys(bridgeTokens ?? []).some(address => {
-    const token = bridgeTokens?.[address]
-    return token?.listIds.has(tokenList?.id)
-  })
+  const isActive = useMemo(() => {
+    if (!bridgeTokens) {
+      return false
+    }
+    return Object.keys(bridgeTokens).some(address => {
+      const token = bridgeTokens[address]
+      return token?.listIds.has(tokenList?.id)
+    })
+  }, [bridgeTokens])
```

Comment:
That would be nice to refactor that a bitt, but maybe that's out of scope of that PR.